_candidates_en/YoshitakaOkuda_1.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Yoshitaka Okuda
+title: "Lesson learned on Flow API implementation through development of ScalikeJDBC streams"
```

Comment:
Consider changing to ""Lessons learned on Flow API implementation through development of ScalikeJDBC streams""